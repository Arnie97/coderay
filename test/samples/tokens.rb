require 'coderay'

p CodeRay.scan("puts 3 + 4, '3 + 4'", :ruby)
