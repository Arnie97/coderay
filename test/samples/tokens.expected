[["puts", :ident], [" ", :space], ["3", :integer], [" ", :space], ["+", :operator], [" ", :space], ["4", :integer], [",", :operator], [" ", :space], [:open, :string], ["'", :delimiter], ["3 + 4", :content], ["'", :delimiter], [:close, :string]]
